pnpm-workspace.yaml

```diff
@@ -42,10 +43,10 @@ catalogs:
     vite-plugin-inspect: ^11.0.0
     vue: ^3.5.13
     vue-tsc: ^2.2.8
-
```

Comment:
/cc @antfu 

pnpm does autoformats like this when using e.g. `pnpm patch`. Maybe we should leave it like this to prevent future noise/merge conflicts.